ReleaseHistory.md

```diff
@@ -2,6 +2,9 @@
 ## UNRELEASED
 * DEP: Remove explicit versioning for `System.Memory` and `System.Runtime.CompilerServices.Unsafe`.
 * DEP: Remove spurious references to `System.Collections.Immutable`.
+* DEP: Update `Microsoft.Data.SqlClient` reference from 2.1.2 to 2.1.7 in `WorkItems` and `Sarif.Multitool.Library` to resolve [CVE-2024-0056](https://github.com/advisories/GHSA-98g6-xh36-x2p7).
+* DEP: Update `System.Data.SqlClient` reference from 4.8.5 to 4.8.6 in `WorkItems` to resolve [CVE-2024-0056](https://github.com/advisories/GHSA-98g6-xh36-x2p7).
+* BUG: Fix the issue `Stack` object doesn't create corresponding `PhysicalLocation` from stack trace if it contains relative file path.
```

Comment:
>' [](http://example.com/codeflow?start=41&length=1)

Your release note doesn't include information on what helper was literally updated. Please include this information.

Your method is `StackFrame.Create`. Please try to provide this level of specificity for a change like this. You have told customers 'i updated the stack object' but a stronger statement is 'i fixed this method on this class'. 

> * BUG: Update `Stack.Create` method to properly create `PhysicalLocation` instances when stack frames reference relative file paths.

Reply:
updated according to offline chat.